OpenRA.Mods.Common/Traits/TooltipDescription.cs

```diff
@@ -21,7 +21,7 @@ public class TooltipDescriptionInfo : ConditionalTraitInfo
 		public readonly string Description = "";
 
 		[Desc("Player stances who can view the description.")]
-		public readonly Stance ValidStances = Stance.Ally | Stance.Neutral | Stance.Enemy;
+		public readonly Stance ValidStances = Stance.Enemy | Stance.Neutral | Stance.Ally;
```

Comment:
This ordering is bad. :S

Reply:
The previous order caused my IDE to show an error so I changed it to the order used in the Enum, see https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Traits/TraitsInterfaces.cs#L52.

Reply:
Fixed the stance order.